src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -17,17 +20,22 @@ export enum AdminVenueTab {
   run = "run",
 }
 
+export interface AdminVenueViewRouteParams {
+  venueId?: string;
+  selectedTab?: AdminVenueTab;
+}
+
 const adminVenueTabLabelMap: Readonly<Record<AdminVenueTab, String>> = {
   [AdminVenueTab.spaces]: "Spaces",
   [AdminVenueTab.timing]: "Timing",
   [AdminVenueTab.run]: "Run",
 };
 
-const DEFAULT_TAB = AdminVenueTab.spaces;
-
 export const AdminVenueView: React.FC = () => {
-  const venueId = useVenueId();
```

Comment:
NB: `useVenueId()` functionality is already covered by `useParams()` and the only difference of handling legacy `?venueId=` query parameters shouldn't be an issue for the new Admin NG